src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,38 @@
+.watch {
+  @include clock;
+  background-color: black;
+  width: $watch-size;
+  height: $watch-size;
+  margin-top: 50px;
+  border: 4px solid $main-color;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+
+  &__center {
+    @include design;
+    @include clock;
+    height: $center-size;
+    width: $center-size;
+  }
+
+  &__seconds {
+    @include design;
```

Comment:
design mixin and position mixin both have a position:absolute rule, try to not repeat it